js/src/forum/components/Notification.js

```diff
@@ -2,6 +2,7 @@ import Component from '../../common/Component';
 import avatar from '../../common/helpers/avatar';
 import icon from '../../common/helpers/icon';
 import humanTime from '../../common/helpers/humanTime';
+import Button from "../../common/components/Button";
```

Comment:
Quotes :scream:

Reply:
I guess it's time to add linting to ci 🙈